frontend/.env.template

```
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_API_BASE_URL=http://localhost:9000
```

```suggestion
API_BASE_URL=http://localhost:9000
```
aspidaがfetchするのは現状server actionだからブラウザ側にバックエンドサーバーURL露出させなくていいかも